libcodechecker/analyze/tu_collector.py

```diff
@@ -0,0 +1,257 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+import codecs
+import json
+import logging
+import os
+import random
+import re
+import shlex
+import string
+import subprocess
+import zipfile
+
+
+def __random_string(l):
+    return ''.join(random.choice(string.ascii_lowercase) for i in range(l))
+
+
+def __get_toolchain_compiler(command):
+    tcpath = None
+
+    for cmp_opt in command:
+        if '--gcc-toolchain' in cmp_opt:
+            tcpath = \
```

Comment:
I think `tcpath` variable can be eliminated because we can return from the function here.